### PR TITLE
pass 'source' as a query param

### PR DESCRIPTION
**Pull request by camhashemi**

### CODE REVIEW

File: kifi-backend/modules/shoebox/angular/src/keep/keepService.js [hunk 1]
```diff
@@ -17,7 +17,7 @@ angular.module('kifi')
     var api = {
       addMessageToKeepDiscussion: function (keepId, text) {
         return net
-        .addMessageToKeepDiscussion(keepId, { text: text, source: 'Kifi.com' }) // '/kifi-backend/modules/common/app/com/keepit/discussion/MessageSource'
+        .addMessageToKeepDiscussion(keepId, 'Kifi.com', { text: text }) // '/kifi-backend/modules/common/app/com/keepit/discussion/MessageSource'
```
Comment (andrewconner) [inline kifi-backend/modules/shoebox/angular/src/keep/keepService.js:20]: Couldn't you tell this from the backend endpoint?

Comment (camhashemi) [inline kifi-backend/modules/shoebox/angular/src/keep/keepService.js:20]: We've moved over to shoebox for messaging on the ext (still need to release a new version there, but the code is in master). We can use `USER_AGENT` perhaps, but this param was being passed to eliza.

Comment (andrewconner) [inline kifi-backend/modules/shoebox/angular/src/keep/keepService.js:20]: Shouldn't the endpoints be different? i.e., site, extension, mobile all using different shoebox endpoints.

Comment (camhashemi) [inline kifi-backend/modules/shoebox/angular/src/keep/keepService.js:20]: ah, right. The API is mostly the same, but it makes sense for versioning and whatnot. will do.

**Pull request closed without merging**